vignettes/magclass.Rmd

```diff
@@ -140,6 +140,11 @@ rename region set in "zones"
 getSets(pm)[2] <- "year"
 ```
 
+Finally, to get a quick impression of the content of the object, you can get a rough plot:
```

Comment:
@pascal-sauer Maybe you could have another, brief look at this? I have just added it.